tests/resources/word_bank.csv

```diff
@@ -1,6 +1,6 @@
-word_id;Deutsch;Deutscher Ausdruck 1;Deutscher Ausdruck 2;Deutscher Ausdruck 3;Deutscher Ausdruck 4;Spanisch;Spanischer Ausdruck 1;Spanischer Ausdruck 2;Spanischer Ausdruck 3;Spanischer Ausdruck 4
-WID1;ab und zu;Vielleicht darf ich ihn mir ab und zu ausleihen;;;;de vez en cuando;Tal vez me lo puedes prestar de vez en cuando.;;;
-WID2;abändern;Niemand will die Genfer Konvention abändern, aber wir können die Konvention ergänzen.;;;;modificar;Nadie quiere modificar el Convenio de Ginebra, pero sí que podemos complementarlo.;;;
-WID3;abdrehen;Wir müssen einen Hahn abdrehen, der ständig offen ist.;;;;cerrar;Es preciso cerrar un grifo permanentemente abierto.;;;
-WID4;abhalten;Trotzdem, selbst das sollte Dich nicht abhalten.;Ich könnte bis in diese Nacht zurück und sie abhalten, mich wegzugeben.;;;impedir/detener;De todas formas, eso no debería detenerte.;Puedo regresar a aquella noche e impedir que me deje.;;
-WID5;abholen;Seine Wäsche steht zum Abholen bereit.;Wenn du mich abholst, bin ich deine beste Freundin.;;;recoger/venir a buscar;Su colada está lista para recoger.;Si me vienes a buscar, seré tu mejor amiga.;;
+word_id;level;Deutsch;Deutscher Ausdruck 1;Deutscher Ausdruck 2;Deutscher Ausdruck 3;Deutscher Ausdruck 4;Spanisch;Spanischer Ausdruck 1;Spanischer Ausdruck 2;Spanischer Ausdruck 3;Spanischer Ausdruck 4
+WID1;intermediate;ab und zu;Vielleicht darf ich ihn mir ab und zu ausleihen;;;;de vez en cuando;Tal vez me lo puedes prestar de vez en cuando.;;;
+WID2;intermediate;abändern;Niemand will die Genfer Konvention abändern, aber wir können die Konvention ergänzen.;;;;modificar;Nadie quiere modificar el Convenio de Ginebra, pero sí que podemos complementarlo.;;;
+WID3;beginner;abdrehen;Wir müssen einen Hahn abdrehen, der ständig offen ist.;;;;cerrar;Es preciso cerrar un grifo permanentemente abierto.;;;
+WID4;advanced;abhalten;Trotzdem, selbst das sollte Dich nicht abhalten.;Ich könnte bis in diese Nacht zurück und sie abhalten, mich wegzugeben.;;;impedir/detener;De todas formas, eso no debería detenerte.;Puedo regresar a aquella noche e impedir que me deje.;;
+WID5;beginner;abholen;Seine Wäsche steht zum Abholen bereit.;Wenn du mich abholst, bin ich deine beste Freundin.;;;recoger/venir a buscar;Su colada está lista para recoger.;Si me vienes a buscar, seré tu mejor amiga.;;
```

Comment:
oh, that's a good point. we could maybe have an "all" level or something like that so is valid for all levels

Reply:
I would say **null** or empty string should mean "valid for all levels". Then no need to assign a level for all current words (although we'll do it anyways but just in case we left some out)